vendor_blueprints/verify_refresh_status.py

```diff
@@ -0,0 +1,102 @@
+import argparse
+import sys
+import pydomo
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_FINAL_STATUS_SUCCESS = 0
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_INVALID_ACCOUNT = 201
+EXIT_CODE_DATASET_NOT_FOUND = 202
+EXIT_CODE_EXECUTION_ID_NOT_FOUND = 203
+EXIT_CODE_FINAL_STATUS_INVALID = 204
+EXIT_CODE_FINAL_STATUS_CANCELLED = 205
+EXIT_CODE_STATUS_INCOMPLETE = 206
```

Comment:
Make sure these exit codes don't conflict with the other exit codes in the other scripts.

Additionally, for consistency across other blueprints, make the status related error codes start at 210.

I found this was easier to maintain by having a separate errors file that every script imports. Not required though. https://github.com/shipyardapp/tableau-blueprints/blob/main/tableau_blueprints/errors.py